src/TouristInfoCards.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+const TouristInfoCards = props => {
```

Comment:
This component is named `TouristInfoCards` (plural), but it only actually renders 1 card. Do you think that `TouristInfoCard` (singular) would be a better name?